### PR TITLE
Api 5.0 WP 10 (Dice)

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
I don't completely understand how the value for the slot machine is computed from the symbols, so I didn't add a description. But I don't think that's strictly necessary … (https://t.me/pythontelegrambottalk/152143)

### CODE REVIEW

File: telegram/constants.py [hunk 1]
```diff
@@ -83,6 +83,8 @@
     DICE_DICE (:obj:`str`): '🎲'
     DICE_DARTS (:obj:`str`): '🎯'
     DICE_BASKETBALL (:obj:`str`): '🏀'
+    DICE_FOOTBALL (:obj:`str`): '⚽'
+    DICE_SLOT_MACHINE (:obj:`str`): = '🎰'
```
Comment (Poolitzer) [inline telegram/constants.py:87]: so the = is wrong I think

**Review by Poolitzer — COMMENTED** [review]
one minor thing

File: telegram/constants.py (hunk 1 above)
Comment (Bibo-Joshi) [inline telegram/constants.py:87]:
```suggestion
    DICE_SLOT_MACHINE (:obj:`str`): '🎰'
```

**Review by Poolitzer — APPROVED** [review]
LGTM

**Pull request merged**